src/tests/ftest/osa/osa_online_reintegration.yaml

```diff
@@ -6,7 +6,7 @@ hosts:
     - server-D
   test_clients:
     - client-E
-timeout: 1200
+timeout: 1300
```

Comment:
There seems to be a conflict with the avocado test timeout (1300) being less than the `clush_timeout` (1500)

Reply:
Reduced the clush timeout to 1000.